AllHitAnalyze.py

```
@@ -1,234 +1,234 @@
+				f"EXACT±45ms  {Count(self.accurateRate[3])}  {Percentage(self.accurateRate[3], accurateRateSum)}", 
+				f"Exact±90ms  {Count(self.accurateRate[4])}  {Percentage(self.accurateRate[4], accurateRateSum)}", 
+				f"Great±150ms {Count(self.accurateRate[5])}  {Percentage(self.accurateRate[5], accurateRateSum)}", 
+				f"Right＋250ms {Count(self.accurateRate[6])}  {Percentage(self.accurateRate[6], accurateRateSum)}", 
+				f"Miss > 250ms {Count(self.accurateRate[7])}  {Percentage(self.accurateRate[7], accurateRateSum)}", ],
+			)
+		ax2.text(0.1,0.5,f"EXACT        {Count(sum(self.accurateRate[0:4]))}  " \
+			f"{Percentage(sum(self.accurateRate[0:4]), accurateRateSum)}", 
+			ha='center',va='top',fontsize=12,color='#00B5B5', )
+
+if __name__ == '__main__':
 	HitAnalyze().Show()
```

Ruff (0.14.8)

Undefined name `HitAnalyze`

(F821)